ambari-server/src/main/resources/properties.json

```diff
@@ -189,6 +189,7 @@
         "StackServices/stack_name",
         "StackServices/stack_version",
         "StackServices/service_name",
+        "StackServices/service_category",
```

Comment:
`properties.json` is no longer used, see e77a31ab0a.

Reply:
Didn't know about this change. Thanks for noticing